.circleci/config.yml

```diff
@@ -493,6 +503,8 @@ jobs:
       - GO111MODULE=on
     steps:
       - checkout
+      - build/determinator:
+          paths: "feg lte cwf"
```

Comment:
I think you may need to add `orc8r`. At least in local vms if you change a `mconfig` proto and you don't regenerate them (`./build.py`) it will fail